src/api/app/views/webui/request/_set_bugowner_request_dialog.html.erb

```diff
@@ -5,7 +5,7 @@
         <%= form_tag({:action => "set_bugowner_request"}, :name => 'bugowner') do %>
           <p>
             <%= label_tag(:user, "User:") %><br/>
-            <input id="userbkp" name="user" type="hidden" value="False" />
+            <input id="userbkp" name="user" type="hidden" value="" />
```

Comment:
What's this actually for?

Reply:
This was to make set_bugowner_request work. Before it always sent a user or group with `False` value and it always throws an error message that could not found a user or group called `False`

Reply:
Same explanation as with removing `required_parameters` [here](https://github.com/openSUSE/open-build-service/pull/4668/files#diff-a1bb0bdb9ac85a32ba6d96c4289745fcL262). Only one of the parameters is required.